easybuild/easyconfigs/s/SQLite/SQLite-3.19.3-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,49 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the
+# policy:  http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SQLite'
+version = '3.19.3'
+
+homepage = 'http://www.sqlite.org/'
+
+description = 'SQLite: SQL Database Engine in a C Library'
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# eg. http://www.sqlite.org/2017/sqlite-autoconf-3190300.tar.gz
+source_urls = ['http://www.sqlite.org/2017/']
+version_str = '%%(version_major)s%s00' % ''.join('%02d' %
+              int(x) for x in version.split('.')[1:])
```

Comment:
@JackPerdue please remove this file, now that I've merged the separate `SQLite` PR